graphql/database/missions/MissionModel.js

```diff
@@ -0,0 +1,98 @@
+import BaseModel from '../base/BaseModel'
+import types from '../fieldTypes'
+import tableNames from '../tables'
+import { MISSION } from '../constants'
+import { permissionAuthorizers } from '../../utils/authorization-helpers'
+
+/*
+ * @extends BaseModel
+ */
+class Mission extends BaseModel {
+  static get name() {
+    return MISSION
+  }
+
+  static get hashKey() {
+    return 'id'
+  }
+
+  static get tableName() {
+    return tableNames.missions
+  }
+
+  static get schema() {
+    return {
+      id: types
+        .string()
+        .required()
+        .description(`The unique mission id`), // not sure how to nanoid this value
```

Comment:
You can specify length if you want:
https://joi.dev/api/?v=17.4.1#stringlengthlimit-encoding